technology-considerations/s3-user-access-to-objects.md

```diff
@@ -0,0 +1,121 @@
+# Restricting User Access To S3 Objects
```

Comment:
I'm wondering whether we should move this to the [architecture decision records](https://github.com/nationalarchives/tdr-dev-documentation/tree/master/architecture-decision-records) folder. It depends whether this is still just a spike, or if it's actually a decision that we're pretty sure we're going with.

Maybe see what Paul says, and if he's happy with this approach move it into the ADRs? You'll just need to add a date, and make it clear that we've decided on this option.